src/SWP/Bundle/TemplateEngineBundle/Controller/WidgetController.php

```diff
@@ -166,8 +166,8 @@ public function deleteAction(Request $request, $id)
      *     description="Update single widget",
      *     statusCodes={
      *         201="Returned on success.",
-     *         404="Widget not found",
-     *         422="Widget id is not number",
+     *         404="WidgetModel not found",
+     *         422="WidgetModel id is not number",
```

Comment:
it should be just Widget, end users should not be informed that its WidgetModel, for them it's just a Widget